contracts/vortex/CarbonVortex.sol

```diff
@@ -379,7 +373,6 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
         // cache address checks to save gas
         bool carbonControllerIsNotZero = address(_carbonController) != address(0);
         bool vaultIsNotZero = address(_vault) != address(0);
-        bool oldVortexIsNotZero = address(_oldVortex) != address(0);
 
         // withdraw fees from carbon, vault and old vortex
```

Comment:
old vortex part of the comment can be removed

Reply:
done